### PR TITLE
Display time for groups (fixes #80)

### LINKED ISSUE #80: Add an ability to view duration for test group

### PR DESCRIPTION
**Pull request by baev**
screen shot 2017-02-07 at 22 24 05

### CODE REVIEW

File: allure-web/src/components/tree/TreeView.js [hunk 1]
```diff
@@ -34,10 +37,10 @@ class TreeView extends View {
         this.$(e.currentTarget).parent().toggleClass('node__expanded');
     }
 
-    @on('click .statuses-filter')
+    @on('click .tree__control_statuses')
```
Comment (just-boris) [inline allure-web/src/components/tree/TreeView.js]: according to BEM, you need to use `-` in element names: `.tree__control-statuses`
Then active will be `tree__control-statuses_active`. Easier to understand, what is the element name, what is the modification

File: allure-web/src/components/tree/TreeView.hbs [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
+        <a class="fa fa-filter {{b 'tree' 'control' statuses=true}}" data-tooltip="Filter test cases by statues"></a>
+        <a class="fa fa-info-circle {{b 'tree' 'control' info=true}}" data-tooltip="Toggle groups info"></a>
     </span>
 </h2>
 
 <div class="tree tree_root">
     {{#if items}}
         {{#each items}}
-            {{> tree-group baseUrl=../baseUrl}}
+            {{> tree-group baseUrl=../baseUrl root='root' showGroupInfo=../showGroupInfo}}
```
Comment (just-boris) [inline allure-web/src/components/tree/TreeView.hbs]: `root=true` maybe?

File: allure-web/src/components/tree/tree-group.hbs [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {{#if (eq type 'TestGroupNode')}}
-    <div class="node">
+    <div class="{{b 'node'}}">
```
Comment (baev) [inline allure-web/src/components/tree/tree-group.hbs]: ?

**Comment by baev** [comment]
retest this please

**Comment by baev** [comment]
retest this please

**Pull request merged**